src/diracx/db/sql/jobs/schema.py

```diff
@@ -92,6 +92,7 @@ class Jobs(JobDBBase):
     AccountedFlag = Column(
         "AccountedFlag", Enum("True", "False", "Failed"), default="False"
     )
+    VO = Column("VO", String(64), default="Unknown")
```

Comment:
I think this is already done in this PR, which is ready to be merged by the way: https://github.com/DIRACGrid/diracx/pull/102/files